tests/test_coco_and_via_conversions.py

```diff
@@ -0,0 +1,342 @@
+import datetime
+import os
+from pathlib import Path
+
+import pytest
+
+from bboxes_labelling.annotations_utils import (
+    combine_multiple_via_jsons,
+    convert_via_json_to_coco,
+    read_json_file,
+)
+
+
+@pytest.fixture()
+def via_json_1() -> str:
+    """Return path to a sample VIA JSON file.
+
+    Returns
+    -------
+    str
+        path to a sample VIA JSON file 1.
+    """
+    # Return path to sample VIA (Visual Image Annotator) JSON file 1
+    return str(Path("tests/data/COCO_VIA_JSONS/VIA_JSON_1.json").resolve())
```

Comment:
I think we don't need tests in the path because we are in tests directory?
`return str(Path("data/COCO_VIA_JSONS/VIA_JSON_1.json").resolve())`